modules/ROOT/pages/create-instance-task.adoc

```diff
@@ -1,30 +1,38 @@
-= Creating an API instance on API Manager
+= Creating an API instance
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In API Manager 2.x, you can use instances of an API in multiple environments, and you can have multiple instances in the same environment. You can create one instance of an API that serves as a proxy. You can create another to manage directly as a basic endpoint. You can apply caching policies, for example to the proxy, and throttling policies and security policies to the basic endpoint.
+In API Manager 2.x, you can use instances of an API in multiple environments. Additionally, you can also have multiple instances in the same environment. 
 
-After setting up an API version for management, you repeat the process to create another instance. Attempting to set up the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them:
+You can create one instance of an API that serves as a proxy. You can create another to manage directly as a basic endpoint. You can apply caching policies, for example to the proxy, and throttling policies and security policies to the basic endpoint.
 
-image::instances.png[]
+After setting up an API version for management, you repeat the process to create another instance. Attempting to set up the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
 
-. Set up an API to manage from Exchange.
+To create a API instance:
+
+. Select *API Administration > Manage API* to create an API instance from Exchange.
 +
-For example, set up version 1.0 of an API named jsonplaceholder.
+If you attempt to create two API verions of the same name, an instance of the existing API is automatically created. For example, set up version 1.0 of an API named jsonplaceholder.:
 +
 image::manage-exchange-end-task.png[]
 +
-. Repeat the process to set up an instance:
+. From *API Name*, select the same API from Exchange, for example, jsonplaceholder.
+. From *API Version*, select the same version of the API, for example 1.0.
+. From *Asset Version*, select the Exchange asset version.
+. From *Managing Type*, choose Basic Endpoint or Endpoint with Proxy. 
++
+For example, select Endpoint With Proxy.
+. In *Implementation URI*, enter the URL that hosts your web site or an arbitrary URL, such as the example shown `+http://api.domain.com+`. 
++
+An entry, any entry, is required if you chose Endpoint with Proxy. You cannot accept the default example; otherwise, saving this configuration is disabled.
```

Comment:
If you choose *Endpoint with Proxy* and do not make an entry (such as accepting the default value), you cannot save your configuration.

Reply:
Done! Thanks!